vocabularies/geometry-roles.ttl

```diff
@@ -13,70 +13,59 @@
 
 <http://linked.data.gov.au/org/gsq>
     a sdo:Organization ;
     sdo:name "Geological Survey of Queensland" ;
-    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> ;
-.
+    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
 
 grole:centroid a skos:Concept ;
     dct:source <https://en.wikipedia.org/wiki/Centroid> ;
     skos:altLabel "barycenter"@en ;
     skos:definition "The arithmetic mean position of all the Points in a Feature's Geometry, expressed as a single Point."@en ;
     skos:hiddenLabel "center of mass"@en ;
     skos:inScheme <http://linked.data.gov.au/def/geometry-roles> ;
-    skos:prefLabel "Centroid"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/geometry-roles> .
+    skos:prefLabel "Centroid"@en .
 
 grole:bounding-box a skos:Concept ;
-    skos:broader grole:boundary ;
     dct:source <https://en.wikipedia.org/wiki/Minimum_bounding_box> ;
     skos:altLabel "minimum bounding box"@en , "smallest bounding box"@en , "enclosing box"@en , "bounding rectangle"@en ;
     skos:definition "The box (rectangle) with the smallest area that contains all the Points in a Feature's Geometry."@en ;
     skos:inScheme <http://linked.data.gov.au/def/geometry-roles> ;
     skos:prefLabel "Bounding box"@en .
 
 grole:bounding-circle a skos:Concept ;
-    skos:broader grole:boundary ;
     dct:source <https://en.wikipedia.org/wiki/Smallest-circle_problem> ;
     skos:definition "The circle with the smallest area that contains all the Points in a Feature's Geometry."@en ;
     skos:inScheme <http://linked.data.gov.au/def/geometry-roles> ;
     skos:prefLabel "Bounding circle"@en .
 
-grole:boundary a skos:Concept ;
-    skos:definition "A polygon which denotes the extent of a Feature's Geometry."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/geometry-roles> ;
-    skos:prefLabel "Boundary"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/geometry-roles> .
-
 grole:concave-hull a skos:Concept ;
-    skos:broader grole:boundary ;
     skos:definition "The minimum bounding Polygon encompassing all the Points in a Features Geometry, based on a user-defined distance threshold for vectors between Points."@en ;
     skos:inScheme <http://linked.data.gov.au/def/geometry-roles> ;
     skos:prefLabel "Concave hull"@en .
 
 grole:convex-hull a skos:Concept ;    
     dct:source <https://www.arcgis.com/home/item.html?id=a6129839e96d4c0682250673cd89589d> ;
-    skos:broader grole:boundary ;
     skos:definition "The minimum bounding Polygon encompassing all the all Points in a Feature's Geometry."@en ;
     skos:inScheme <http://linked.data.gov.au/def/geometry-roles> ;
     skos:prefLabel "Convex hull"@en .
 
 grole:detailed-geometry a skos:Concept ;
     skos:definition "A Feature's Geometry in detail. It may be regular or irrigular but it does not constitute a Boundary for the Feature."@en ;
```

Comment:
```suggestion
    skos:definition "A Feature's Geometry in detail. It may be regular or irregular. A detailed geometry is an exact representation, or close approximation of the features true shape. Distinct from a boundary, which is a generalised representation."@en ;
```

Reply:
added the words 'enclosing the feature' for clarity. a centroid is a generalised representation  but is not a boundary
```suggestion
    skos:definition "A Feature's Geometry in detail. It may be regular or irregular. A detailed geometry is an exact representation, or close approximation of the features true shape. A detailed geometry may correspond to any geometry type (e.g. point, line, polygon) appropriate to the specific feature being described at the scale of representation. Distinct from a boundary, which is a generalised representation enclosing the feature."@en ;
```